### PR TITLE
improve mass response to work with nodal masses

### PR DESCRIPTION
**Pull request by armingeiser**

### CODE REVIEW

**Comment by armingeiser** [comment]
@philbucher Thanks for the hint, i am reusing part of the `TotalStructuralMassProcess` now

**Comment by philbucher** [comment]
I exposed the `CalculateElementMass` fct as a static function, this way I don't have to set the `NODAL_MASS` on the elements any more (which I recently introduced but didn't like it myself too much) => saving memory

@loumalouomega since both @armingeiser and I pushed to this branch, can you please have a final look?
Thx!

File: applications/StructuralMechanicsApplication/tests/test_mass_calculation.py [hunk 1]
```diff
@@ -282,9 +282,12 @@ def test_solid_mass(self):
         self.assertAlmostEqual(sum(expected_elemental_masses_by_id.values()), total_mass)
 
     def __CheckElementslMasses(self, model_part, expected_elemental_masses_by_id):
```
Comment (armingeiser) [inline applications/StructuralMechanicsApplication/tests/test_mass_calculation.py]: typo, maybe it can be fixed in this PR

**Review by loumalouomega — APPROVED** [review]
Looks OK

**Pull request merged**